extensions/adobe/experience/decisioning/proposition-action.schema.json

```diff
@@ -26,6 +26,14 @@
           "title": "label",
           "description": "The customer generated display name of the element with which the end-user interacted.",
           "type": "string"
+        },
+        "xdm:tokens": {
+          "title": "tokens",
+          "type": "array",
+          "description": "An array of decision item tokens for tracking",
```

Comment:
Let’s use a more solution-agnostic description consistent with the description for id and labels, e.g. “Application-generated metadata associated with the element with which the end-user interacted.”

Reply:
Done.